github-runner-manager/requirements.txt

```diff
@@ -6,4 +6,4 @@ openstacksdk==4.5.0
 pydantic < 2
 pymongo==4.13.0
 click==8.2.1
-flask==3.1.1
+flask==3.1.1
```

Comment:
Why remove the newline at the end of line?

Reply:
by accident